includes/Jobs/RequestWikiRemoteAIJob.php

```diff
@@ -0,0 +1,395 @@
+			return $this->handleDryRun( $outcome, $comment );
+		} else {
+			return $this->handleLiveRun( $outcome, $comment );
+		}
+	}
+
+	private function handleDryRun( string $outcome, string $comment ): bool {
+		$services = MediaWikiServices::getInstance();
+		$outcomeMessage = Message::newFromKey( "requestwikiqueue-$outcome" )->text();
+		$commentText = Message::newFromKey( 'requestwiki-ai-decision-dryrun' )
+		->params( $outcomeMessage, $comment )
+		->inLanguage( $services->getContentLanguage() )
+		->text();
+
+		$this->wikiRequestManager->addComment(
+			comment: $commentText,
+			user: User::newSystemUser( 'CreateWiki AI' ),
+			log: true,
+			type: 'comment',
+			notifyUsers: []
+		);
+
+		$dryRunMessages = [
+			'approve' => "Wiki request {$this->id} was approved by AI but not automatically created.",
+			'moredetails' => "Wiki request {$this->id} needs revision but was not automatically marked.",
+			'decline' => "Wiki request {$this->id} was declined by AI but not automatically marked.",
+			'onhold' => "Wiki request {$this->id} requires manual review.",
+		];
+
+		$this->logger->debug(
+			"DRY RUN: " . ( $dryRunMessages[$outcome] ?? "Unknown outcome for request {$this->id}." ),
+			[
+				'id' => $this->id,
+				'reasoning' => $comment,
+			]
+		);
+
+		return true;
+	}
+
+	private function handleLiveRun( string $outcome, string $comment ): bool {
+		$services = MediaWikiServices::getInstance();
+		$systemUser = User::newSystemUser( 'CreateWiki AI' );
+		$commentText = Message::newFromKey( "requestwiki-ai-decision-$outcome" )
+		->params( $comment )
+		->inLanguage( $services->getContentLanguage() )
+		->text();
+
+		switch ( $outcome ) {
+			case 'approve':
+				$this->wikiRequestManager->startQueryBuilder();
+				$this->wikiRequestManager->approve(
+					user: $systemUser,
+					comment: $commentText
+				);
+				$this->wikiRequestManager->tryExecuteQueryBuilder();
+				$this->logger->debug( "Request {$this->id} auto-approved by AI.\nReason: $comment" );
+				break;
+
+			case 'moredetails':
+				$this->wikiRequestManager->startQueryBuilder();
+				$this->wikiRequestManager->moredetails(
+					user: $systemUser,
+					comment: $commentText
+				);
+				$this->wikiRequestManager->tryExecuteQueryBuilder();
+				$this->logger->debug( "Request {$this->id} requires more details.\nReason: $comment" );
+				break;
+
+			case 'decline':
+				$this->wikiRequestManager->startQueryBuilder();
+				$this->wikiRequestManager->decline(
+					user: $systemUser,
+					comment: $commentText
+				);
+				$this->wikiRequestManager->tryExecuteQueryBuilder();
+				$this->logger->debug( "Request {$this->id} declined by AI.\nReason: $comment" );
+				break;
+
+			case 'onhold':
+			default:
+				$this->wikiRequestManager->addComment(
+					comment: $commentText,
+					user: $systemUser,
+					log: false,
+					type: 'comment',
+					notifyUsers: []
+				);
+				$this->logger->debug( "Request {$this->id} queued for manual review." );
```

Comment:
I mentioned something about the format to use for this in an earlier comment. This and all other logers should use that.